activerecord/lib/active_record/connection_adapters/postgresql/database_statements.rb

```diff
@@ -148,6 +148,37 @@ def build_explain_clause(options = [])
           "EXPLAIN (#{options.join(", ").upcase})"
         end
 
+        # Set when constraints will be checked for the current transaction.
+        #
+        # Not passing any specific constraint names will set the value for all deferrable constraints.
+        #
+        # [<tt>deferred</tt>]
+        #   Valid values are +:deferred+ or +:immediate+.
```

Comment:
```suggestion
        #   Valid values are +:deferred+ or +:immediate+.
        #
```

I'd probably add a break here between the options and the link